test/parallel/test-tls-client-auth.js

```diff
@@ -0,0 +1,293 @@
+'use strict';
+
+require('../common');
+const fixtures = require('../common/fixtures');
+
+const {
+  assert, connect, keys
+} = require(fixtures.path('tls-connect'));
```

Comment:
I don't know. It predates test/common, AFAICT, but perhaps it should have been moved there when test/common was created?  Moving it would be a good code-n-learn activity.